packages/wonder-blocks-typography/src/components/text.js

```diff
@@ -0,0 +1,344 @@
+// @flow
+import * as React from "react";
+
+import {css} from "aphrodite";
+import type {StyleType} from "@khanacademy/wonder-blocks-core";
+
+const TypeMetrics = {
+    Sans: {
+        Family: '"Lato", "Noto Sans", sans-serif',
+        Weights: {
+            Regular: 400,
+            Strong: 700,
+        },
+    },
+    Serif: {
+        Family: '"Source Serif Pro", "Noto Serif", serif',
+        Weights: {
+            Regular: 400,
+            Strong: 600,
+        },
+    },
+    Monospace: {
+        Family: '"Inconsolata", monospace',
+        Weights: {
+            Regular: 400,
+            Strong: 700,
+        },
+    },
+};
+
+const styleFragments = {
+    Common: {
+        MozOsxFontSmoothing: "grayscale",
+        WebkitFontSmoothing: "antialiased",
+        display: "block",
+        margin: 0,
+    },
+    SerifRegular: {
+        fontFamily: TypeMetrics.Serif.Family,
+        fontWeight: TypeMetrics.Serif.Weights.Regular,
+    },
+    SerifStrong: {
+        fontFamily: TypeMetrics.Serif.Family,
+        fontWeight: TypeMetrics.Serif.Weights.Strong,
+    },
+    SansRegular: {
+        fontFamily: TypeMetrics.Sans.Family,
+        fontWeight: TypeMetrics.Sans.Weights.Regular,
+    },
+    SansStrong: {
+        fontFamily: TypeMetrics.Sans.Family,
+        fontWeight: TypeMetrics.Sans.Weights.Strong,
+    },
+    MonospaceRegular: {
+        fontFamily: TypeMetrics.Monospace.Family,
+        fontWeight: TypeMetrics.Monospace.Weights.Regular,
+    },
+    MonospaceStrong: {
+        fontFamily: TypeMetrics.Monospace.Family,
+        fontWeight: TypeMetrics.Monospace.Weights.Strong,
+    },
+    // Heading
+    Heading: {
+        display: "block",
+    },
+    HeadingXXXLarge: {
+        fontSize: "60px",
+        lineHeight: "68px",
+    },
+    HeadingXXLarge: {
+        fontSize: "48px",
+        lineHeight: "54px",
+    },
+    HeadingXLarge: {
+        fontSize: "38px",
+        lineHeight: "42px",
+    },
+    HeadingLarge: {
+        fontSize: "30px",
+        lineHeight: "34px",
+    },
+    HeadingMedium: {
+        fontSize: "24px",
+        lineHeight: "28px",
+    },
+    HeadingSmall: {
+        fontSize: "20px",
+        lineHeight: "22px",
+    },
+    HeadingXSmall: {
+        fontSize: "12px",
+        lineHeight: "16px",
+    },
+    // Compact Heading
+    CompactHeading: {
```

Comment:
I remember seeing this in the Figma specs. In which situations do you think we would need to use `CompactHeading`?